client/me/get-apps/index.jsx

```diff
@@ -23,7 +24,7 @@ export const GetApps = () => {
 			<MeSidebarNavigation />
 			<GetAppsIllustration />
 			<MobileDownloadCard />
-			<DesktopDownloadCard />
+			{ ! config( 'env_id' ).startsWith( 'desktop' ) && <DesktopDownloadCard /> }
```

Comment:
if we included whether or not we're building desktop in the webpack defines we could skip this conditional in the output build, though since it's actually the web that needs it and desktop that doesn't then I don't think it warrants much attention

I verified that we have `desktop` and `desktop-development` environments which should both be picked up here.